articles/hooks/guides/use-the-credentials-exchange-extensibility-point.md

```diff
@@ -1,49 +1,22 @@
 ---
-title: Using the Credentials Exchange Extensibility Point
-description: The credentials-exchange extensibility point for use with Hooks
-toc: true
+description: How to use the credentials exchange extensibility point for use with Hooks
 beta: true
 topics:
     - hooks
     - extensibility-points
     - credentials-exchange
 contentType:
   - how-to
-  - concept
 useCase: extensibility-hooks
+v2: true
 ---
+# Use the Credentials Exchange Extensibility Point
```

Comment:
My preference would be to name this topic: "Change Scopes and Add Custom Claims to Access Tokens." If there are other ways to do that, maybe it should be mentioned in and then linked from different concept doc that covers all the ways. Then the title of this one could be more specific but it would be super long - which isn't good either.  Possibly: "Change Scopes and Add Custom Claims Using Credentials Exchange." This is a tricky one.